images/base/files/usr/local/bin/entrypoint

```diff
@@ -154,6 +154,33 @@ select_iptables() {
   update-alternatives --set ip6tables "/usr/sbin/ip6tables-${mode}" > /dev/null
 }
 
+enable_network_magic(){
```

Comment:
maybe a more informative name :P (and move magic to a comment)
`replace_docker_dns` ?